src/native/clr/include/xamarin-app.hh

```diff
@@ -394,8 +394,7 @@ struct MarshalMethodName
 
 [[gnu::visibility("default")]] extern const char* const mm_class_names[];
 [[gnu::visibility("default")]] extern const MarshalMethodName mm_method_names[];
+#endif // def RELEASE
 
 using get_function_pointer_fn = void(*)(uint32_t mono_image_index, uint32_t class_index, uint32_t method_token, void*& target_ptr);
-
-[[gnu::visibility("default")]] extern void xamarin_app_init (JNIEnv *env, get_function_pointer_fn fn) noexcept;
-#endif // def RELEASE
+extern "C" [[gnu::visibility("default")]] void xamarin_app_init (JNIEnv *env, get_function_pointer_fn fn) noexcept;
```

Comment:
Why was this moved outside of the `#if RELEASE` block?  Is it somehow present in Debug config builds? (Are those even meaningfully supported at present?)

Reply:
…or maybe it always needs to exist because it's in the invoke-tables…